### DISCUSSION TITLE
I NEED 1.20.6🙏🙏🙏

### DISCUSSION THREAD

**Discussion by Draken-arch** [Faboslav/friends-and-foes#256] [Q&A]
FABOSLAV, I call apon thee. Please make a 1.20.6 version please! I have a really nice quality of life modpack and I have a world with nearly 200 days worth of work, And I saw your mod in a youtube video and I want to add it to my modpack, however you dont have a 1.20.6 version for FORGE and thats what my modpacks on. So if you ever see this please make a 1.20.6 version. I REPETE I NEED A 1.20.6 VERSION

**Comment by digi2303** [comment]
Uhm: https://modrinth.com/mod/friends-and-foes/version/fabric-mc1.20.6-2.0.17

**Comment by digi2303** [comment]
You should use this or NeoForge.
